### PR TITLE
Add forward upstream cluster wellknown name

### PR DESCRIPTION
**Pull request by EItanya**
Update wellknown names of more filters

### CODE REVIEW

**Comment by stevesloka** [comment]
I think this duplicates https://github.com/envoyproxy/go-control-plane/pull/294

**Comment by EItanya** [comment]
> I think this duplicates #294

Indeed, completely missed that, sorry!

**Comment by EItanya** [comment]
@stevesloka any idea when that other PR will be merged, it seems like it's been open for quite a while. Also, I reopened this one with just one new wellknown name, but the check doesn't seem to be rerunning

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "go.inferGopath": false
```
Comment (kyessenov) [inline .vscode/settings.json:2]: Can we remove this file from the commit?

**Comment by kyessenov** [comment]
Folks, you don't have to specify the name to apply the extension. Starting early this year, envoy relies on the type URL of the extension config to select an extension.

**Pull request closed without merging**